rules/solid/tests/Rector/Variable/MoveVariableDeclarationNearReferenceRector/Fixture/variable_in_closure.php.inc

```diff
@@ -16,7 +16,7 @@ function variableInClosure()
 
 function variableInClosure()
 {
-    $closure = function () {
+    return function () {
```

Comment:
This is just for testing purposes, it had no effect. The original test case is covered here:
https://github.com/rectorphp/rector/pull/4410/commits/97a3004237cf923373bceb0da3d6ad749ce13bc2#diff-7cabfedb3086541a6f5995aed5be677cc009cb92d09cd5560ede872dc4f67947